### PR TITLE
Feature - Refine directory structure

### PR DESCRIPTION
**Pull request by hnq90**
In this PR, we changed directories' structure:
- Add `app` directory for php application.
- Add `data` for logs/data.
- Add `services` for Docker containers.
- Update usage of `fix_permission` scripts.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ script:
   - docker-compose up -d --build
   - docker-compose ps
   # change permission for writable directories
-  - docker exec -it dockerforphp_php7_1 /bin/bash /app/fix_perms.sh /app/storage
+  - docker exec -it dockerforphp_php7_1 /usr/local/bin/fix_permission.sh /app/storage
```
Comment (oanhnn) [inline .travis.yml]: Chuyển thành như sau thì hay hơn
```
docker exec -it dockerforphp_php7_1 /usr/local/bin/fix_permission /app/storage
```

File: README.md [hunk 2]
```diff
@@ -6,33 +6,36 @@ A stack for running PHP7 application inside Docker containers.
...
 
 ```bash
 $ git clone git@github.com:rikkeisoft/docker-for-php.git deploy
 $ docker-compose up -d
 ```
 
 If you have problem with file permission, please ensure you set correct the owner, which PHP-FPM is running under, in `php-fpm.conf`. Then execute:
+
+```bash
+$ docker exec -it dockerforphp_php7_1 /usr/local/bin/app/fix_permission.sh -u <your_php_fpm_user> <your_path>
```
Comment (oanhnn) [inline README.md]:
```
docker exec -it dockerforphp_php7_1 /usr/local/bin/fix_permission /app/storage
```

File: services/php7.0-fpm/Dockerfile [hunk 3]
```diff
@@ -20,6 +20,8 @@ RUN apt-get update && apt-get install -y \
 
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 
+COPY fix_permission.sh /usr/local/bin/fix_permission.sh
```
Comment (oanhnn) [inline services/php7.0-fpm/Dockerfile]:
```
COPY fix_permission.sh /usr/local/bin/fix_permission
```

**Review by oanhnn — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (hnq90) [reply .travis.yml]: Done :+1:

File: README.md (hunk 2 above)
Comment (hnq90) [reply README.md]: Done :+1:

File: services/php7.0-fpm/Dockerfile (hunk 3 above)
Comment (hnq90) [reply services/php7.0-fpm/Dockerfile]: Done :+1:

**Review by oanhnn — APPROVED** [review]

**Pull request merged**